linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_configure_remote_server/oval/shared.xml

```diff
@@ -0,0 +1,34 @@
+<def-group>
+  <definition class="compliance" id="auditd_audispd_configure_remote_server" version="1">
+    <metadata>
+      <title>Configure audispd Plugin Remote Server IP address or Hostname</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <description>remote_server setting in /etc/audisp/audisp-remote.conf is set to a certain IP address or hostname</description>
+    </metadata>
+    <criteria>
+        <criterion comment="remote_server setting in /etc/audisp/audisp-remote.conf" test_ref="test_auditd_audispd_configure_remote_server" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" comment="remote server to send audit records" id="test_auditd_audispd_configure_remote_server" version="1">
+    <ind:object object_ref="object_auditd_audispd_configure_remote_server" />
+    <ind:state state_ref="state_auditd_audispd_configure_remote_server" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object id="object_auditd_audispd_configure_remote_server" version="1">
+    <ind:filepath>/etc/audisp/audisp-remote.conf</ind:filepath>
+    <!-- Allow only space (exactly) as delimiter: https://fedorahosted.org/audit/browser/trunk/src/auditd-config.c#L426 -->
```

Comment:
This link goes to fedorahosted retired page.

Reply:
Nice catch! It's there 12 times in multiple rules

Reply:
I would prefer to remove the link because the place in code can easily change over time and we don't have capacity to watch the changes in auditd sources.

Reply:
@yuumasato Do you like the solution that I removed the link? If yes, could I open a separate PR that removes the remaining occurrences from other OVALs?